tests/ha/fencing.pm

```diff
@@ -32,8 +32,8 @@ sub run {
         # Fence the node
         assert_script_run 'crm -F node fence ' . get_node_to_join;
 
-        # Wait a little to be sure that fence command is on his way
```

Comment:
line 27-28 - saying us that you need first reset consoles and only then wait for barrier. Is it wrong ? please change comment or move reset_console before waiting for barrier

Reply:
Before fencing but on all nodes except 2nd node that execute the fencing. I will change the comment, or remove it if the simplification discuss above works.